draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -959,37 +976,37 @@ For a given request ("request"), the following algorithm returns `same-site` or
 2.  Let `site` be `request`'s client's "site for cookies" (as defined in the
     following sections).
 
-3.  Let `target` be the registrable domain of `request`'s current url.
+3.  Let `target` be the origin of `request`'s current url.
 
-4.  If `site` is an exact match for `target`, return `same-site`.
+4.  If `site` is same-site with `target`, return `same-site`.
 
 5.  Return `cross-site`.
 
-The request's client's "site for cookies" is calculated depending upon its
-client's type, as described in the following subsections:
+The request's client's "site for cookies" is either a triple origin or a
+globally unique identifier. It is calculated depending upon its client's type,
+as described in the following subsections:
```

Comment:
There is some overlap with https://github.com/whatwg/html/pull/4966. No need to do anything now, but when we publish this PR, I think it's worth pointing this out and whether we should align things. (Maybe a request client has a top-level origin plus a "same-site with ancestors" boolean to handle the recursive ancestry business.)

Though it's unclear to me whether this is defining it for a document or a request's client or what.

Reply:
I'll make a note during the offical PR.